components/server/src/api/server.ts

```diff
@@ -101,10 +104,22 @@ export class API {
         return {
             get(target, prop) {
                 return (...args: any[]) => {
-                    const method = type.methods[prop as any];
+                    const logContext: LogContextOptions & {
+                        requestId?: string;
+                        contextTimeMs: number;
+                        grpc_service: string;
+                        grpc_method: string;
+                    } = {
+                        contextTimeMs: performance.now(),
```

Comment:
I'm capturing it here because otherwise i cannot handle async generators propely